letta/server/rest_api/routers/v1/agents.py

```diff
@@ -474,7 +472,11 @@ async def send_message_to_agent(
 
             # return a stream
             return StreamingResponse(
-                sse_async_generator(streaming_interface.get_generator(), finish_message=include_final_message),
+                sse_async_generator(
```

Comment:
FYI the way this works is in the generator yield code we pass in the task reference, which we use to get the return value for `server.send_messages` (which is type `LettaUsageStatistics`).